cfme/common/__init__.py

```diff
@@ -102,13 +102,6 @@ class Validatable(object):
     """
     property_tuples = []
 
-    def properties_ui(self):
```

Comment:
@jkrocil Thank you! Extending `Validatable` from `SummaryPage` is ok? or shall I merge both classes into one?